lib/xcake/command.rb

```diff
@@ -5,12 +5,17 @@ class Command < CLAide::Command
 
     self.command = 'xcake'
     self.version = VERSION
-    self.description = 'Create and maintain Xcode project files easily.'
+    self.description = 'Create and maintain Xcoe project files easily.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.